vas.sh

```
@@ -146,6 +146,12 @@ get_version() {
     fi
 }
 
+get_user_id() {
+    local_container=$1
```

Shellcheck

[warning] local_container appears unused. Verify use (or export if used externally).

(SC2034)